### PR TITLE
Add Calendly Oauth2 docs

### PR DESCRIPTION
**Pull request by Joffcom**
Adds OAuth2 docs for Calendly

### CODE REVIEW

File: docs/integrations/builtin/credentials/calendly.md [hunk 1]
```diff
@@ -30,3 +31,24 @@ To configure this credential, you'll need:
...
+To create a new OAuth app after logging in to [Calendly's developer portal](https://developer.calendly.com/console/apps){:target=_blank .external-link}, go to **Account > My Apps > Create new app**.
+
+Use these settings:
+
+ - Give your app a name like **n8n Automation**
+ - Set **Kind of app** to **Web**
+ - Set **Environment type** to **Sandbox** or **Production**
+ - Copy the **OAuth Redirect URL** from n8n and enter it as a **Redirect URI** in the OAuth app.
+ - Copy the **Client ID** for the Calendly app and enter this as your n8n **Client ID**.
+ - Copy the **Client secret** from Calendly and enter this as your n8n **Client Secret**
```
Comment (ayatnkw) [inline docs/integrations/builtin/credentials/calendly.md:51]:
```suggestion
To set up the integration: 

 1. Give your app a name like **n8n Automation**.
 2. Set **Kind of app** to **Web**.
 3. Set **Environment type** to **Sandbox** or **Production**.
 4. Copy the **OAuth Redirect URL** from n8n and enter it as a **Redirect URI** in the OAuth app.
 5. Copy the **Client ID** for the Calendly app and enter this as your n8n **Client ID**.
 6. Copy the **Client secret** from Calendly and enter this as your n8n **Client Secret**.
```

Comment (ayatnkw) [reply docs/integrations/builtin/credentials/calendly.md:51]: Changed the formatting to match the new template @freakwriter @Joffcom

**Review by freakwriter — COMMENTED** [review]
@ayatnkw beat me to the suggested format changes; once that change is committed, I'll happily approve. :)

**Comment by Joffcom** [comment]
Thanks for fixing this @ayatnkw I can't approve my own PR though so you would need to approve it so we can merge :)

**Review by ayatnkw — APPROVED** [review]

**Pull request merged**